.ci/run.sh

```diff
@@ -9,10 +9,39 @@ set -e
 cidir=$(dirname "$0")
 source "${cidir}/lib.sh"
 
+install_docker_ubuntu() {
+	pkg_name="docker-ce"
+	repo_url="https://download.docker.com/linux/ubuntu"
+	curl -fsSL "${repo_url}/gpg" | sudo apt-key add -
```

Comment:
These options look inconsistent to me:

```
 -f, --fail          Fail silently (no output at all) on HTTP errors
 -S, --show-error    Show error even when -s is used
 -s, --silent        Silent mode
```

Why are you suppressing errors anyway as that will make debugging a remote CI very hard?

Reply:
thanks @jodh-intel -, I fold the change into a new file, also tweak this curl command.